style/sass/.scss-lint.yml

```diff
@@ -181,6 +181,11 @@ linters:
     enabled: true
     style: one_space
 
+  SpaceAfterComment:
+    enabled: true
+    style: one_space
```

Comment:
Yes, and yes it does not like SassDoc’s `///` syntax. But we don’t use SassDoc on everyday projects, just Bourbon and Neat and other libraries. So you can disable it specifically when using SassDoc.